core/input/input.h

```diff
@@ -327,11 +383,30 @@ class Input : public Object {
 	String get_joy_guid(int p_device) const;
 	void set_fallback_mapping(String p_guid);
 
+	// DO NOT call force_flush_buffered_events() in normal course of events,
+	// as it will break agile input on any frame it is called.
+	// Instead rely on flush_buffered_events_tick() and
+	// flush_buffered_events_frame() which are called from
+	// Main::iteration().
+	void force_flush_buffered_events();
```

Comment:
Is this mostly kept for backward compatibility? Can it be made `private` to prevent its use within the codebase?

Reply:
On reflection I decided to remove this and reinstated the old `flush_buffered_events()` function to always call a flush, and avoided calling it in the mainloop on Android. This should be backward compatible while still allowing agile input on platforms that support it.